cpp/modules/deck.gl/layers/src/line-layer/line-layer.cpp

```diff
@@ -158,7 +159,7 @@ auto LineLayer::_getModel(wgpu::Device device) -> std::shared_ptr<lumagl::Model>
                                      fs,
                                      attributeSchema,
                                      instancedAttributeSchema,
-                                     {{sizeof(LineLayerUniforms)}, {sizeof(ViewportUniforms)}}};
+                                     {{sizeof(ViewportUniforms)}, {sizeof(LineLayerUniforms)}}};
```

Comment:
These sizeofs are both 256, due to alignas directive? Worth verifying by logging?